frontend/homework/8/handson.ts

```diff
@@ -0,0 +1,67 @@
+
+
+
+interface Recipe {
+    image: string;
+    name: string;
+    rating: number;
+    cuisine: string;
+    ingredients: string[];
+    difficulty: string;
+    prepTimeMinutes: number;
+    cookTimeMinutes: number;
+    caloriesPerServing: number;
+}
+
+interface RecipeResponse {
+    recipes: Recipe[];
+}
+
+
+async function fetchRecipesFromAPI(): Promise<RecipeResponse> {
+    const response = await fetch('https://dummyjson.com/recipes');
+    const jsonresponse = await response.json();
+    return jsonresponse as RecipeResponse;
+}
+
+
+async function printAllRecipes(recipes : RecipeResponse): Promise<void> {
+    
+    var count=1;
```

Comment:
Why do we need this explicit count variable?

Reply:
I did it just to segregate all recipes for better readability on console.